functions/v2/cloud-event-logging/index.js

```diff
@@ -0,0 +1,40 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+// [START functions_structured_logging_event]
+const {Logging} = require('@google-cloud/logging');
+const functions = require('@google-cloud/functions-framework');
+const pkg = require('./package.json');
+
+functions.cloudEvent('structuredLoggingEvent', async () => {
+  // Initialize the logging client
+  const logging = new Logging();
+  // Create a LogSync transport, defaulting to process.stdout
+  const log = logging.logSync(pkg.name);
+  // Required to capture your project id
+  await logging.setProjectId('gromit-project-2');
+
+  const text = 'Hello, world!';
+  const entry = log.entry(
+    {
+      severity: 'NOTICE',
+      component: 'arbitrary-property'
```

Comment:
@pattishin does this look like a linting issue you should fix?

Reply:
sure does :)